rosdep/python.yaml

```diff
@@ -893,7 +897,7 @@ python-mako:
   ubuntu: [python-mako]
 python-mapnik:
   fedora: [mapnik-python]
-  ubuntu: [python-mapnik2]
+  ubuntu: [python-mapnik]
```

Comment:
`python-mapnik` should work for all versions. `python-mapnik2` only exists for Trusty and there it's just a transitional package that depends on `python-mapnik`